modules/features/player/src/main/java/au/com/shiftyjelly/pocketcasts/player/view/PlayerContainerFragment.kt

```diff
@@ -208,6 +258,35 @@ class PlayerContainerFragment : BaseFragment(), HasBackstack {
             menuRes = null,
             fragmentManager = parentFragmentManager,
         )
+
+
+        viewModel.networkType.observe(viewLifecycleOwner) { type ->
+            when (type) {
+
+                ConnectivityManager.TYPE_WIFI -> {
+                    Timber.tag("Check Data Connectivity").d("Connect to wifi 😊")
+                    println("Connect to wifi ")
+
+                }
+
+                ConnectivityManager.TYPE_MOBILE -> {
+                    //Log Purposes
+                    Timber.tag("Check Data Connectivity").d("Connect to Mobile Internet 😊")
+                    println("Connect to Mobile Internet ")
+                    //Toast when user is using Mobile Internet
+                    Toast.makeText(context, "Using Mobile Data", Toast.LENGTH_SHORT).show()
```

Comment:
This is showing the text anytime the network type changes, even if playback is not currently happening, whereas [the linked issue](https://github.com/Automattic/pocket-casts-android/issues/1374) is just addressing the situation where someone is actively streaming an episode and they switch to mobile data and they don't get any warning even though they have turned on the setting for showing a warning when streaming mobile data. Basically, I think we only want to respond when the user has that setting turned on and the user is streaming an episode that is switching to use mobile data.